### DISCUSSION TITLE
Displaying array of saving sessions

### DISCUSSION THREAD

**Discussion by djdarlo** [BottlecapDave/HomeAssistant-OctopusEnergy#632] [Q&A]
Does anyone have a smart way of displaying the array of sessions returned by the saving sessions events?

**Comment by BottlecapDave** [comment]
Hello. You can probably modify [this example](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/discussions/528#discussioncomment-7569088) to achieve what you want. You put the code inside a markdown card within the dashboard.
